src/components/views/rooms/RoomHeader.tsx

```
@@ -150,6 +159,97 @@ interface VideoCallButtonProps {
     behavior: DisabledWithReason | "legacy_or_jitsi" | "element" | "jitsi_or_element";
 }
 
+function QRCodeModal({ show, onClose, qrData, checkbox,room,destination,amount,currency }) {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)